ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/writers/TomlEmitter.kt

```diff
@@ -207,6 +207,32 @@ public abstract class TomlEmitter(config: TomlConfig) {
      */
     public fun emitValue(boolean: Boolean): Unit = emit(boolean.toString())
 
+    /**
+     * Emits an [Instant] value.
+     *
+     * @param instant
+     */
+    public fun emitValue(instant: Instant): Unit = emit(instant.toString())
+
+    /**
+     * Emits a [LocalDateTime] value.
+     *
+     * @param dateTime
+     */
+    public fun emitValue(dateTime: LocalDateTime): Unit = emit(dateTime.toString())
```

Comment:
all these methods do the same - convert value `toString`, may be unify it with generic method?

Reply:
Well, a public generic method would allow passing potentially invalid values (the emitter API is public after all). The only common class between these is `Comparable`. Does that matter?

A private method like

```kotlin
private fun emitValue(value: Any) = emit(value.toString())
```

is doesn't have these issues though.

Reply:
yeah, ok, let it be :)